frontend/src/InteractiveImport/Movie/SelectMovieModalContent.tsx

```diff
@@ -172,9 +172,11 @@ function SelectMovieModalContent(props: SelectMovieModalContentProps) {
 
     return sorted.filter(
       (item) =>
-        item.title.toLowerCase().includes(filter.toLowerCase()) ||
-        item.tmdbId.toString().includes(filter) ||
-        item.imdbId?.includes(filter)
+        item.sortTitle.includes(
+          filter.toLowerCase().replace(/[^a-zA-Z ]/g, '')
+        ) ||
+        item.tmdbId.toString() === filter ||
+        item.imdbId === filter
```

Comment:
Is imdbId still being used? I believe you removed the link in #191

Reply:
Sorry, must have spaced on that.  Will revise it this weekend and push.  IIRC, I had a hard time completely removing it, and undid that just to get this fix submitted, since the fix isn't technically related to removing the IMDB tech debt.

Reply:
No worries, just an observation. Wanted to make sure, we can clean up as we go.

Reply:
Removed IMDB from here.  I did not go on a mission to find any and all traces in other files, but at least it's gone from the ones I've touched for this fix.